pep-0698.rst

```diff
@@ -3,6 +3,7 @@ Title: Override Decorator for Static Typing
 Author: Steven Troxler <steven.troxler@gmail.com>,
         Joshua Xu <jxu425@fb.com>,
         Shannon Zhu <szhu@fb.com>
+Discussions-To: typing-sig@python.org
 Status: Draft
 Sponsor: Jelle Zijlstra <jelle.zijlstra at gmail.com>
```

Comment:
```suggestion
Sponsor: Jelle Zijlstra <jelle.zijlstra at gmail.com>
Discussions-To: https://mail.python.org/archives/list/typing-sig@python.org/thread/TOIYZ3SNPBJZDBRU3ZSBREXV2NNHF4KW/
Status: Draft
```

* Link the actual canonical discussion thread, not just the name of the email list, per PEP 12 guidance
* Put the headers in the standard canonical order, per the [PEP template](https://peps.python.org/pep-0012/#suggested-sections)

Reply:
Should header ordering be covered by the linter?

I think it's out of order on existing peps (I used 612 as an example). Seems like a better thing to automate than manually check against the template. It probably wouldn't be too hard to also update all existing PEPs.

Reply:
> I think it's out of order on existing peps (I used 612 as an example).

612 has the correct order (as do all but three other PEPs), but its best to use the current PEP template in PEP 12 as a template, since many technical and formatting practices have evolved over time, particularly recently.

> Should header ordering be covered by the linter?

It should be, yeah; right now the linters are just a bunch of handwritten, overly complex regexes I wrote rather than an actual program (unlike the actual header parsing code that renders the PEPs), and its not totally trivial to do. However, after some effort to get it working, I added it in #2835 and fixed the three PEPs that didn't conform, thanks.